src/ledger/insert_shred.zig

```diff
@@ -762,6 +823,8 @@ pub const ShredInserter = struct {
         slot: Slot,
         parent_slot: Slot,
     ) !*SlotMetaWorkingSetEntry {
+        // QUESTION: Any reason why no metrics is being recorded here, but it is
+        // for getIndexMetaEntry with index_meta_time_us
```

Comment:
There's not a very good reason. I just copied agave. There are actually a few similar ones also that could be measured, but are not.